src/js/dragging.js

```diff
@@ -0,0 +1,82 @@
+export function drag() {
+    let leftPane = document.getElementById('left_pane')
+    let rightPane = document.getElementById('right_pane')
+    let separator = document.getElementById('separator')
+
+    separator.style.cursor = 'move'
+
+    let leftLimit = 10
+    let rightLimit = 90
+
+    function sdrag(onDrag, onStop, direction) {
+        let startX = 0
+        let startY = 0
+        let el = separator
+        let dragging = false
+
+        function dragstart() {
+            dragging = true
+        }
+    
+        function dragmove(e) {
+            if (dragging) {
+                let fix = {}
+                    onDrag && onDrag(el, e.pageX, startX, e.pageY, startY, fix)
```

Comment:
Please use `if` to for check instead of `onDrag && ...`
Also correct tabulation of this and next lines